packages/core/src/accordion/accordion-panel.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
```

Comment:
I think this file was a bad rebase, it no longer exists on next.

Reply:
I wondered, I got it backwards then. Will clean these up.